### PR TITLE
Add more ops and todos for missing ops

### PR DESCRIPTION
**Pull request by sbrunk**

### CODE REVIEW

**Comment by sbrunk** [comment]
> Hi @sbrunk - Apologies for initially contacting you to contribute to storch and then disappearing in a long hiatus; I am now finally free to contribute.
> 
> I can help implement some of the items in your TODO list, I will start implementing some of them on a branch forking this one.

Awesome!

>  Some functions, such as `softmax` are implemented in different modules and exported in `torch`. I noticed PyTorch documentation states that some of the `torch.*` functions are aliases from other modules, are you thinking Storch should keep a similar structure?

Yes I'm trying to keep the package structure similar to upstream PyTorch. I hope this makes it easier to find and keep up to the original implementations. Feel free to improve on that if you have suggestions. I might not have been entirely consistent.

>  Even though this module contains thin wrapper for torch functions, I assume tests will be useful on storch end to make sure things are wrapped properly. I would happily add tests for each, and could help me understand each of them, but wouldn't do it if you think its not needed.

Yes please do! Having more tests would be super helpful.
A good source for simple test cases are the examples in the Python docs.
